content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md

```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+title: Debug Init Containers
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página mostra como investigar problemas relacionados à execução de
+inicialização de Contêineres. As linhas de comando de exemplo abaixo 
+referem-se ao pod como `<pod-name>` e a inicialização de contêineres como `<init-container-1>` e
+`<init-container-2>`.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Você deve estar familiarizado com os fundamentos da
+  [inicialização de Contêineres](/docs/concepts/workloads/pods/init-containers/).
+* Você deve ter [Configurado uma Inicialização de Contêiner](/docs/tasks/configure-pod-container/configure-pod-initialization/#creating-a-pod-that-has-an-init-container/).
+
+<!-- steps -->
+
+## Verificando o status das inicializações de contêineres
```

Comment:
```suggestion
## Verificando o status das contêineres de inicializações
```

Reply:
I think this suggestion mostly works, except that "inicializações" should be singular.